atc/db/migration/migrations/1569945021_add_components.up.sql

```diff
@@ -0,0 +1,31 @@
+BEGIN;
+
+  CREATE TABLE components (
+      id serial PRIMARY KEY,
+      name text NOT NULL,
+      interval text NOT NULL,
+      last_ran timestamp WITH TIME ZONE,
+      paused boolean DEFAULT FALSE
+  );
+
+  CREATE UNIQUE INDEX components_name_key ON components (name);
+
+  INSERT INTO components(name, interval) VALUES
```

Comment:
Can/should we avoid this by turning the `UpdateIntervals` into an upsert/sync of all components? I didn't expect to maintain these as part of a migration. When we add a new migration would we need a migration to add the component?

Reply:
yup good point, fixed in ef0fd2251332c7099be15e2e0498e87196dcfccd